CONTRIBUTING.md

```diff
@@ -14,32 +14,49 @@ forward, please consider contributing in the following areas:
 
 ## Development Setup
 
-To get started hacking on mitmproxy, please install a recent version of Python (we require at least Python 3.8). The
-following commands should work on your system:
+To get started hacking on mitmproxy, please install a recent version of Python (we require at least Python 3.8). 
+Then, do the following:
 
+##### Linux / macOS
 ```shell
+# 1) Verify that these commands work:
 python3 --version
 python3 -m pip --help
 python3 -m venv --help
+# 2) Install:
+git clone https://github.com/mitmproxy/mitmproxy.git
+cd mitmproxy
+python3 -m venv venv
+venv/bin/pip install -e .[dev]
```

Comment:
Why did you choose to deep-ref the pip binary, instead of activating the venv?

Reply:
I feel that it's simpler this way because we don't need to handle virtualenv activation here and then again at the end of the section.